articles/active-directory/develop/active-directory-saml-claims-customization.md

```diff
@@ -53,8 +53,9 @@ From the **Choose name identifier format** dropdown, you can select one of the f
 |---------------|-------------|
 | **Default** | Microsoft identity platform will use the default source format. |
 | **Persistent** | Microsoft identity platform will use Persistent as the NameID format. |
-| **EmailAddress** | Microsoft identity platform will use EmailAddress as the NameID format. |
+| **Email address** | Microsoft identity platform will use EmailAddress as the NameID format. |
```

Comment:
@AllisonAm Can you point me to this drop-down in the portal:

> To view or edit the claims issued in the SAML token to the application, open the application in Azure portal. Then open the **User Attributes & Claims** section.

I was going to verify the naming, but I can't seem to find what the article describes.

Here's what I have:

![image](https://user-images.githubusercontent.com/13394541/135899014-468a132b-7ed4-480b-bfac-64c56c2b1d1a.png)

**Token configuration** has optional claims, and I can select SAML, but I can't configure what is described in this article.